gender_novels/novel.py

```diff
@@ -145,6 +145,7 @@ def _load_novel_text(self):
             start_novel = text.find('***', end_intro_boilerplate + 5) + 3
             end_novel = text.find('*** END OF THIS PROJECT GUTENBERG EBOOK')
             text = text[start_novel:end_novel]
+            text = text.replace('\n', ' ')
```

Comment:
Looks good to me, but does this break anyone else's work?  I think there are a few tests that this breaks.